hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/response/ContractCallResponse.java

```diff
@@ -39,4 +42,14 @@ public static String convertContractCallResponseToSelector(final ContractCallRes
     public static String convertContractCallResponseToAddress(final ContractCallResponse response) {
         return Bytes.fromHexString(response.getResult()).slice(12).toUnprefixedHexString();
     }
+
+    public static String hexToASCII(String str) throws DecoderException {
```

Comment:
Would prefer these two new methods as non-static methods.

```suggestion
    public String getResultAsText() throws DecoderException {
```